nestkernel/syn_id_delay.h

```diff
@@ -34,12 +34,12 @@ struct SynIdDelay
 {
   unsigned int delay : NUM_BITS_DELAY;
   unsigned int syn_id : NUM_BITS_SYN_ID;
-  bool subsequent_targets : 1;
+  bool more_targets : 1;
```

Comment:
This should be initialized with `true` since it is a `bool`.

Reply:
i think this should be initialized with `false` and this is done in https://github.com/nest/nest-simulator/blob/ebae2906f82fe3893e68ab48d1f123f8865cadc2/nestkernel/syn_id_delay.h#L42. Am I missing something here?

Reply:
Sorry, I had misread the code, all is fine.